src/components/dashboard/components/dictionary/common/DictionaryModal.jsx

```diff
@@ -12,7 +13,7 @@ export class DictionaryModal extends React.Component {
     this.state = {
       data: {
         id: '',
-        preffered_sources: '',
+        preffered_source: '',
```

Comment:
I changed it because the data that returns from the server says `preferred_source` and it is just `string`, also there was a ticket specifying that it should be preferred source. Yes, multiple sources will be added later but I think it is single for now and also to avoid confusion.